lib/osf-components/addon/components/responsive-dropdown/template.hbs

```diff
@@ -9,7 +9,7 @@
     as |dd|
 >
     {{yield (hash
-        trigger=(component dd.Trigger defaultClass=(local-class 'Trigger'))
+        trigger=(component dd.Trigger defaultClass=(local-class (if @buttonStyling 'Button')))
```

Comment:
Do you need an else 'Trigger' on this styling? It looks like you're losing the defaultClass you had before.

Reply:
The `.Trigger` style was only there to add `outline: 0` onfocus, so I just removed it completely